pkgs/applications/science/biology/seaview/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchurl, pkgs }:
+
+stdenv.mkDerivation rec {
+  version = "4.7";
+  name = "seaview-${version}";
+
+  src = fetchurl {
+    url = "ftp://pbil.univ-lyon1.fr/pub/mol_phylogeny/seaview/archive/seaview_${version}.tar.gz";
+    sha256 = "0fhyq7dcn0izhwcfin9ajsr7kmmsqm9f1np1rmhzg4digfwqb29n";
+  };
+
+  buildInputs = with pkgs; [ fltk libjpeg ];
+
+  patchPhase = "sed -i 's#PATH=/bin:/usr/bin#/usr/bin/env#' seaview.cxx";
+  installPhase = "mkdir -p $out/bin; cp seaview $out/bin";
+
+  meta = with stdenv.lib; {
+    description = "GUI for molecular phylogeny";
+    longDescription = ''
+SeaView is a multiplatform, graphical user interface for multiple sequence alignment and molecular phylogeny.
```

Comment:
Would you mind indenting this? The multiline strings strip common leading whitespace just fine.

Reply:
Not at all. Will indent in the next iteration.